docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -50,7 +50,7 @@ Replicas count towards this limit, but closed indexes do not. An index with 5
 primary shards and 2 replicas will be counted as 15 shards. Any closed index
 is counted as 0, no matter how many shards and replicas it contains.
 
-The limit defaults to 1,000 shards per data node, and can be dynamically
+The limit defaults to 1,000 shards per data node, this includes open indices with unassigned primary or replica shards, and can be dynamically
 adjusted using the following property:
```

Comment:
I'd combine the info about what counts toward the limit with the info in the previous paragraph. Maybe:

The cluster shard limit defaults to 1,000 shards per data node. Both primary and replica shards of all open indices count toward the limit, including unassigned shards. For example, an open index with 5 primary shards and 2 replicas counts as 15 shards. Closed indices do not contribute to the shard count. You can dynamically adjust the limit with the following property: